src/app/components/server/Icons.tsx

```diff
@@ -364,3 +364,30 @@ export const PasswordIcon = (
     </svg>
   );
 };
+
+// Keywords: time, estimated, watch
+export const ClockIcon = (
+  props: SVGProps<SVGSVGElement> & { alt?: string }
```

Comment:
Like https://github.com/mozilla/blurts-server/pull/3355 :)

```suggestion
  props: SVGProps<SVGSVGElement> & { alt: string }
```